log-manager/src/main/java/io/airlift/log/LoggingMBean.java

```diff
@@ -15,58 +15,77 @@
  */
 package io.airlift.log;
 
-import ch.qos.logback.classic.Logger;
-import ch.qos.logback.classic.LoggerContext;
-import org.slf4j.LoggerFactory;
 import org.weakref.jmx.Managed;
 
+import java.util.Collections;
 import java.util.Map;
+import java.util.TreeMap;
+import java.util.logging.*;
+import java.util.logging.Logger;
 
-import static ch.qos.logback.classic.Level.toLevel;
-import static com.google.common.collect.Maps.newTreeMap;
-import static org.slf4j.Logger.ROOT_LOGGER_NAME;
-
+@SuppressWarnings("MethodMayBeStatic")
```

Comment:
I guess I was lazy.  All of the methods can be static because they use `Logging` which is static.  I'll just tag the methods.